src/components/nowPlayingBar/nowPlayingBar.js

```diff
@@ -146,13 +157,38 @@ define(['require', 'datetime', 'itemHelper', 'events', 'browser', 'imageLoader',
             }
         });
 
-        elem.querySelector('.previousTrackButton').addEventListener('click', function () {
+        elem.querySelector('.previousTrackButton').addEventListener('click', function (e) {
+            if (currentPlayer) {
+                if (currentPlayer.id === 'htmlaudioplayer' && (currentPlayer._currentTime >= 5 || !playbackManager.previousTrack(currentPlayer))) {
+                    // Cancel this event if doubleclick is fired
+                    if (e.detail > 1 && playbackManager.previousTrack(currentPlayer)) {
+                        return;
+                    }
+                    playbackManager.seekPercent(0, currentPlayer);
+                    // This is done automatically by playbackManager, however, setting this here gives instant visual feedback
```

Comment:
@MrTimscampi The default for the dblclick JS event. We can set our own, but IMO it's fine and on par with what you would expect in all other music apps.

For context, the delay it has is the same as the one you have when double tapping on any mobile browser and it zooms.

Check https://developer.mozilla.org/en-US/docs/Web/API/Element/dblclick_event example also.

Reply:
Hm, it's nearly instantaneous here. It should be mostly fine if we let playbackManager handle the position instead of forcing it.

Reply:
@MrTimscampi My bad, I thought you meant the back behaviour. Here is a video in Chrome (maybe in FF it's faster and you didn't notice it, I can say in Chrome it is). Maybe I'm too nitpicky, but I'm able to tell the difference easily

Both videos hit back at the 1:40 timestamp. Bear in mind with the low framerates of gifs, it's not the best way to show this here, to be honest.

**With "manual" change:**
![after](https://user-images.githubusercontent.com/10274099/85418971-ca688200-b571-11ea-802a-d1f69700c2e5.gif)

**With ``playbackmanager``**:
![before](https://user-images.githubusercontent.com/10274099/85419272-2af7bf00-b572-11ea-8f97-8ef4297a6461.gif)

Reply:
Hm, I think the difference is negligible and it's better to not have stuff interacting with something that's controlled in a central component without going through that component, imo.

It's not blocking, but I think it's better design-wise to let the playbackManager handle the seekbar :p

Reply:
@MrTimscampi I'll add a TODO so we can investigate later the delay that playbackManager has in the seek function. I'm full for this because makes the difference between a webpage and the "sensation" that we're in a native app, which I think Android users will appreciate.